x/distribution/types/params_internal_test.go

```diff
@@ -0,0 +1,34 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func Test_validateAuxFuncs(t *testing.T) {
+	type args struct {
+		i interface{}
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{"wrong type", args{10.5}, true},
+		{"nil Int pointer", args{sdk.Dec{}}, true},
```

Comment:
shouldn't we call this test `"empty sdk.Dec"`?

Reply:
make a suggestion and I'll accept it

Reply:
ok